### ISSUE TITLE
Set proxy to remote browser

### ISSUE THREAD

**Issue by sorokin-ihor** [go-rod/rod#1127]
Rod Version: latest

is it possible to use a remote browser multithreaded and at the same time install a proxy in it? i am using the latest version of image on an ubuntu server. Docker

**Comment by ysmood** [comment]
Yes, the same as https://github.com/go-rod/rod/blob/0096e531a49c82f272f5661eabe755d2e7c1eb61/examples_test.go#L391

**Comment by sorokin-ihor** [comment]
> Yes, the same as
> 
> https://github.com/go-rod/rod/blob/0096e531a49c82f272f5661eabe755d2e7c1eb61/examples_test.go#L391

in this case? 
`func main() {
	u := launcher.MustResolveURL("http://")
	
	browser := rod.New().ControlURL(u).MustConnect()

	fmt.Println(
		browser.MustPage("https://example.com").MustEval("() => document.title"),
	)
}
`

**Comment by ysmood** [comment]
You have to install the proxy on the remote server.

**Comment by sorokin-ihor** [comment]
> You have to install the proxy on the remote server.

Ahaha, really? Then why am I asking the question here?

**Comment by ysmood** [comment]
This is due to a limitation of chromium cdp protocol which is control by chrome team, there's no easy way to fix right now.
